xs2a-adapter-aspsp-registry/src/test/java/de/adorsys/xs2a/adapter/registry/mapper/AspspMapperTest.java

```diff
@@ -0,0 +1,26 @@
+package de.adorsys.xs2a.adapter.registry.mapper;
+
+import de.adorsys.xs2a.adapter.registry.AspspCsvRecord;
+import de.adorsys.xs2a.adapter.service.model.Aspsp;
+import org.junit.Test;
+import org.mapstruct.factory.Mappers;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AspspMapperTest {
+
+    private final AspspMapper aspspMapper = Mappers.getMapper(AspspMapper.class);
+
+    @Test
+    public void toAspspCsvRecord() {
+        Aspsp aspsp = new Aspsp();
+        aspsp.setId("42");
+        aspsp.setAdapterId("test-adapter");
+
+        AspspCsvRecord aspspCsvRecord = aspspMapper.toAspspCsvRecord(aspsp);
+
+        assertThat(aspspCsvRecord).isNotNull();
+        assertThat(aspspCsvRecord).hasFieldOrPropertyWithValue("id", "42");
```

Comment:
I propose to not to use `hasFieldOrPropertyWithValue()` method if `getters` are available for the performance reasons. As far as I see from this method arguments, I'm pretty sure it uses Reflection API under the hood. But, as `AspspCsvRecord` class contains `getters`, why don't we use them in such a way:
```
assertThat(aspspCsvRecord.getId()).isEqualTo("42");
```
Also, this `42` and `test-adapter` could be extracted to the class constants (`private static final String`).

What do you think?

Reply:
Good point, thanks for the comprehensive comment!